package.json

```diff
@@ -0,0 +1,147 @@
+        }
+      }
+    ],
+    "rules": {
+      "react/forbid-prop-types": "off"
+    },
+    "parser": "babel-eslint",
+    "settings": {
+      "polyfills": [
+        "Promise",
+        "Map"
+      ]
+    }
+  },
+  "stylelint": {
+    "extends": "stylelint-config-terra",
+    "rules": {
+      "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
+    }
+  },
+  "scripts": {
+    "bootstrap:hoist": "lerna bootstrap --hoist",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
+    "clean:install": "npm run clean && npm install",
+    "clean:obsolete-snapshots": "npm test -- -u",
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:build": "lerna ls | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:clean": "rm -rf packages/**/lib",
+    "compile:src": "lerna run compile",
+    "compile:prod": "rm -rf ./build; NODE_ENV=production webpack -p",
+    "compile:lowlight": "rm -rf ./build; THEME=clinical-lowlight-theme webpack -p",
+    "compile:fusion": "rm -rf ./build; THEME=orion-fusion-theme webpack -p",
+    "danger": "danger ci",
+    "dependency-markdown": "node scripts/dependency-markdown-generator/DependencyMarkdownGenerator.js",
+    "docs": "node scripts/root-readme-updater/index.js",
+    "docs:peerDependencies": "node scripts/peer-dependency-generator/index.js",
+    "heroku-prebuild": "npx lerna init",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
+    "jest": "jest",
+    "jest:coverage": "jest --coverage",
+    "link-parent-bin": "link-parent-bin",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx .",
+    "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
+    "pretest": "npm run lint",
+    "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
+    "start": "tt-serve",
+    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
+    "start-prod": "tt-serve --env.disableHotReloading -p",
+    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "test": "npm run lint && npm run jest && npm run wdio",
+    "wdio-default": "wdio",
+    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
+    "wdio-fusion": "THEME=orion-fusion-theme wdio",
+    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion",
+    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.5.0",
+    "@babel/core": "^7.5.0",
+    "@babel/plugin-proposal-class-properties": "^7.4.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.5.0",
+    "@babel/plugin-transform-async-to-generator": "^7.5.0",
+    "@babel/plugin-transform-object-assign": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.5.0",
```

Comment:
Do we know we need each of these plugins as this time?

Reply:
We probably don't, they were copied from terra-framework.

Reply:
I removed what seemed to make sense https://github.com/cerner/terra-graphs/pull/1/commits/e9d6aec8c6afbb208a75de90779ea8e7aea00345